sunpy/net/fido_factory.py

```diff
@@ -214,6 +214,50 @@ def show(self, *cols):
         """
         return type(self)(*[i.show(*cols) for i in self._list])
 
+
+    def show_in_notebook(self, **kwargs):
+        """
+        Display the tables in the current object as interactive grids in a Jupyter Notebook.
+
+        This function utilizes the ``ipydatagrid`` library to render tables as interactive grids.
+        Each table in the current object is converted to a ``pandas.DataFrame``, passed to an
+        ``ipydatagrid.DataGrid`` instance, and displayed directly in the notebook.
+
+        .. note::
+            This function requires the optional dependencies ``pandas``, ``IPython`` and ``ipydatagrid``.
+            Ensure these are installed before calling this method.
+
+        Parameters
+        ----------
+        **kwargs : dict, optional
+            Additional keyword arguments to customize the ``ipydatagrid.DataGrid`` instance.
+            For available options, use ``help(ipydatagrid.DataGrid)`` in a Jupyter Notebook.
+
+        Raises
+        ------
+        ValueError
+            If no tables are available to display.
+
+        Returns
+        -------
+        None
+            The method displays the grids directly in the notebook and does not return anything.
+        """
+        from ipydatagrid import DataGrid
+        from IPython.display import display
+
+        if not self._list:
+            raise ValueError("No tables to display.")
```

Comment:
Why should this error?

Reply:
If `_list` is empty, this raises an error to inform the user of this exact issue. Should I remove it, now I think its unnecessary?

Reply:
If its empty, why should the code raise an error?